python/ray/tune/suggest/hyperopt.py

```diff
@@ -151,7 +166,7 @@ def on_trial_complete(self,
         del self._live_trial_mapping[trial_id]
 
     def _to_hyperopt_result(self, result):
-        return {"loss": -result[self._reward_attr], "status": "ok"}
+        return {"loss": self._metric_op * result[self._metric], "status": "ok"}
```

Comment:
careful, as hyperopt automatically minimizes (rather than maximizes). You will need to update the switch case in L81 to reflect this.